### ISSUE TITLE
[Feature Request] 1.18.2 Bee functionality

### ISSUE THREAD

**Issue by Neoteric-Dev** [MarkusBordihn/BOs-Easy-Mob-Farm#79]
Looks like bee functionality does not exist in 1.18.2

**Comment by MarkusBordihn** [comment]
Could you please provide more details about this?  
I recently released some bee-specific enhancements, which you can find here: [Bee Enhancements](https://github.com/MarkusBordihn/BOs-Easy-Mob-Farm/wiki/MobFarm#bee-enhancements).  

Please note that it may take some time for the latest version (8.5.0) to become visible on all platforms.

**Comment by Neoteric-Dev** [comment]
I have placed a bee inside a T3 Bee Hive Farm.
It produces nothing.

I assumed a bee hive farm would still produce honey or honeycomb even without enhancements.

**Comment by MarkusBordihn** [comment]
Thanks for the feedback!

You're right—normally, bees don't drop anything, regardless of how they are killed (by a player, weapon, fire, water, etc.), because they have an empty loot table.

I checked the previous version, and in that version, bees dropped honeycombs regardless of the mob farm setup, thanks to a configurable option.

The main difference was that bees could only be placed in the bee farm. However, in the new version, this restriction has been removed, so bees can now be used in any mob farm.

I really like your idea of giving additional drops when the correct farm is used for a specific mob. 
To address this, I'll add some "bonus" drops for mobs like bees when placed in the "bee farm."

That said, the bee-specific enhancements will always provide more drops than a plain mob farm, even after the "bonus" drops are implemented.

Does this align better with how you think the farm should work? Let me know!
